fitbenchmarking/resproc/visual_pages.py

```diff
@@ -9,6 +9,7 @@
 import os
 
 from fitbenchmarking.utils.logging_setup import logger
```

Comment:
flake8 complains that logger is imported and not used (not your fault, but might as well fix it here)

Reply:
removed logger